### PR TITLE
nettraffic: Add version 1.64.0

### LINKED ISSUE #3460: Add NetTraffic
[NetTraffic](https://www.venea.net/web/nettraffic_portable) is an essential network bandwidth usage monitor.
It seems can not download from outside, someone can help with this?

The [download page](https://www.venea.net/web/downloads_start/nettraffic_portable) says:
> If the download doesn't start automatically, please use this direct link. Do not use any downloader plugin or external downloading software. Disable page translation.

### PR DESCRIPTION
**Pull request by wenmin92**
- Closes #3460 

[NetTraffic](https://www.venea.net/web/nettraffic): An essential network bandwidth usage monitor.

**Note:** 
- The official site does not provide a valid download url. So I open an issue #3460, thanks to @issaclin32, I use [TechSpot](https://www.techspot.com/downloads/6058-nettraffic.html) as the download service. 
- The official site offers a portable version, but we can not get it, so just use the installer.

### CODE REVIEW

File: bucket/nettraffic.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+    "homepage": "https://www.venea.net/web/nettraffic",
+    "description": "Essential network bandwidth usage monitor",
+    "license": "Freeware",
+    "version": "1.64.0",
+    "url": "https://files02.tchspt.com/storage2/temp/NetTraffic1.64.0.zip",
+    "hash": "0afe87c024a49eca4d141f5903952e0577f79c40a575a6d3f1320e94b67a2025",
+    "pre_install": [
+        "Expand-7zipArchive \"$dir\\NetTraffic 1.64.0.exe\" -Removal",
```
Comment (Ash258) [inline bucket/nettraffic.json]: ...

```suggestion
        "Expand-7zipArchive \"$dir\\NetTraffic*.exe\" -Removal",
```

File: bucket/nettraffic.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\`$LOCALAPPDATA\", \"$dir\\uninstall.exe\" -Force -Recurse"
```
Comment (Ash258) [inline bucket/nettraffic.json]:
```suggestion
        "Remove-Item \"$dir\\`$*\", \"$dir\\uninstall.exe\" -Recurse"
```

File: bucket/nettraffic.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    ],
+    "shortcuts": [
+        [
+            "NetTraffic.exe",
+            "NetTraffic"
+        ]
+    ],
+    "checkver": {
```
Comment (Ash258) [inline bucket/nettraffic.json]:
```suggestion
    "persist": "Data",
    "checkver": {
```

File: bucket/nettraffic.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
+{
+    "homepage": "https://www.venea.net/web/nettraffic",
+    "description": "Essential network bandwidth usage monitor",
+    "license": "Freeware",
```
Comment (Ash258) [inline bucket/nettraffic.json]:
```suggestion
    "license": {
        "identifier": "Freeware",
        "url": "https://www.venea.net/web/nettraffic#license"
    },
```

**Comment by wenmin92** [comment]
I always made a lot of mistakes. 😢

File: bucket/nettraffic.json [hunk 5]
```diff
@@ -0,0 +1,29 @@
+{
+    "homepage": "https://www.venea.net/web/nettraffic",
+    "description": "Essential network bandwidth usage monitor",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://www.venea.net/web/nettraffic#license"
+    },
+    "version": "1.64.0",
+    "url": "https://files02.tchspt.com/storage2/temp/NetTraffic1.64.0.zip",
```
Comment (Ash258) [inline bucket/nettraffic.json]: Is there any proof that this is permalink will last for longer period of time? I pretty doubt based on the `temp` and assumption of load balancer existence on techpost site.

Comment (wenmin92) [reply bucket/nettraffic.json]: Well, up to now, there's no problem and there's no other better method. If someone provides a more reliable link, we can change it at any time.

**Pull request merged**